CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
+* [maestro] Updated maestro to [v2.9.0](https://github.com/PelionIoT/maestro/releases/tag/v2.9.0)
+  * Gateway capabilities - Allows gateways to advertise the features supported by the platform. Maestro utilizes Edge Core's GRM JSON-RPC APIs to add to the gateway device's LwM2M resources. The registered resources are added under Pelion's reserved FeatureMgmt LwM2M object - 33457 with 3 resources - 0 - featureID, 1 - enabled, 2 - config.
+  * Remote config management via LwM2M - FeatureMgmt config resource allows users to remotely view the current configuration of the feature and also push a config update using the LwM2M cloud service APIs. Maestro, on receiving an update, writes the content to the file path specified in the respective parameter config_filepath.
+  * Removed parsing and generation of self-signed certificates. Also removed the platforms rp200 and wwrelay which are no longer supported.
+  * Maestro traffic now routed through edge-proxy instead of directly connecting to the Pelion Cloud
+* [logrotate] Added bbappend file to install a configuration which rotates the logs of files under `/var/log`.
+* [wait-for-pelion-identity] Converted the node program to bash. Removed the generation of self-signed certificate.
+* [edge-proxy] Configured services gateway logs (maestro), gateway stats (maestro), relay-term, fluentbit http endpoint to route traffic through edge-proxy and not consume self-signed certificate.
+* [meta-nodejs] Removed the dependency on node v8.x. Upgraded the node packages to work with default nodejs version of Dunfell.
+* [relay-term] Upgraded relay-term to work with node v12.x. Established an independent recipe from wwrelay-utils and removed the dependency on global-node-modules.
+* [fluentbit] Added recipe to install Fluentbit 1.3.5 on the gateway for providing an open source Log Processor and Forwarder solution.
+  * By default, fluentbit is configured with following input endpoints - CPU, MEM, Systemd services - edge-core, edge-proxy, pelion-relay-term, maestro, kubelet, docker and wait-for-pelion-identity and Tail for /wigwag/log/devicejs.log. And an output endpoint HTTP to publish logs at API endpoint - `http://gateways.local:8080/v3/device-logs` (routing through edge-proxy).
+* [journald] Enabled Forward Secure Sealing (FSS) feature of systemd journal. This will help detect gateway logs file tampering.
+  * To configure Pelion Edge gateway with sealing key and to keep track of verification key in production setup, use Pelion Edge Provisioner (pep) tool [v2.3.0](https://github.com/PelionIoT/pelion-edge-provisioner/releases/tag/v2.3.0).
+  * Provides the means to make logging persistent. Documentation on how to do this is updated.
+* [systemd] Configured to manage the network. By default, disabled Maestro's network management feature.
+* [depreciation] Removed deprecated services.
```

Comment:
Making sure, but we're alright with doing this in a minor release, not a major release, right?

Reply:
Ping @x85446. I think so we are good here because those services were anyways not used in production environment.